### PR TITLE
Update docs.dasch.swiss/community

### PR DESCRIPTION
**Pull request by kilchenmann**
Resolves [DSP-268](https://dasch.myjetbrains.com/youtrack/issue/DSP-268)

### CODE REVIEW

File: docs/community/about-us.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# About us
+
+Since 2017, the Data and Service Center for humanities (DaSCH) has been a member of the Swiss Academy of Humanities and Social Sciences. The main task of the institution is to operate a platform for humanities research data that ensures access to this data. In addition, the networking of data with other databases is to be promoted (linked open data), thus creating added value for research and the interested public.
```
Comment (flavens) [inline docs/community/about-us.md:3]: there is a problem with the full name of the DaSCH here and on the website: the main title is Data and Service Center for **the H**umanities (which I think is the right syntax) and in some paragraph there is  Data and Service Center for humanities -> we must decide and keep the naming consistent!

File: docs/community/product-updates.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
-Product Updates
+# Product Updates
 
+<!-- Here we can probably add the version matrix? -->
+
+## DSP-API (previous knora-api)
```
Comment (flavens) [inline docs/community/product-updates.md:5]: I would write "previously DSP-xx" instead of "previous .."

Comment (kilchenmann) [reply docs/community/product-updates.md:5]: previously knora-xx --> agree 😉

File: docs/community/about-us.md (hunk 1 above)
Comment (kilchenmann) [reply docs/community/about-us.md:3]: I copied from the website 🙈 Data and Service Center for **the** Humanities is the correct syntax. Thanks. I will update both...

**Comment by kilchenmann** [comment]
@flavens I just updated the dasch name everywhere with the correct syntax: Data and Service Center for **the** Humanities

**Pull request merged**